liota/lib/transports/mqtt.py

```diff
@@ -320,12 +323,27 @@ def subscribe(self, topic, qos, callback):
         :return:
         """
         try:
+            if [topic,qos,callback] not in self.sub_list:
+                self.sub_list.append([topic,qos,callback])
             subscribe_response = self._paho_client.subscribe(topic, qos)
             self._paho_client.message_callback_add(topic, callback)
             log.info("Topic subscribed with information: " + str(subscribe_response))
         except Exception:
             log.exception("MQTT subscribe exception traceback..")
 
+    def unsubscribe(self, topic):
```

Comment:
Does anything actually call this method?

Reply:
This is an implementation which I have done in MQTT transports library which was missing previously in Liota. This method can be used by some other DCC`s in order to unsubscribe.